services/QuillLMS/engines/comprehension/lib/comprehension/comprehension/spelling_check.rb

```diff
@@ -27,13 +27,17 @@ def feedback_object
       }
     end
 
+    def non_optimal_feedback_string
+      spelling_rule&.feedbacks&.first&.text || FALLBACK_INCORRECT_FEEDBACK
+    end
+
     def error
       bing_response['error'] ? bing_response['error']['message'] : nil
     end
 
     private def spelling_rule
```

Comment:
Would memoization apply here?
`@spelling_rule ||= Rule.where(rule_type: FEEDBACK_TYPE).first`

Reply:
Sure, that seems reasonable.